### DISCUSSION TITLE
add audio后无法使用

### DISCUSSION THREAD

**Discussion by guguji5** [ZuodaoTech/everyone-can-use-english#848] [Q&A]
环境 macbook m1

我在audio的add resources里添加了 https://github.com/ZuodaoTech/everyone-can-use-english/blob/main/book/files/TOELFL-PartC-93/audios/10.mp3 后，打开如下图所示

<img width="556" alt="image" src="https://github.com/user-attachments/assets/4865a58f-3756-4d0e-88c3-c14d796ae70c">

**Comment by an-lee** [comment]
> 环境 macbook m1
> 
> 我在audio的add resources里添加了 https://github.com/ZuodaoTech/everyone-can-use-english/blob/main/book/files/TOELFL-PartC-93/audios/10.mp3 后，打开如下图所示

因为 https://github.com/ZuodaoTech/everyone-can-use-english/blob/main/book/files/TOELFL-PartC-93/audios/10.mp3 这个链接打开是个网页，并没有下载到真正的音频。你可以试试手动下载音频文件，再从本地添加。
